tests/test_minimal/test_tools/test_importing.py

```diff
@@ -11,5 +11,21 @@ def test_guide_attributes():
     for name, summary in summaries.items():
         for key, value in summary.items():
             assert value is not None, f"{name} is missing GUIDE related attribute {key}."
-            if isinstance(value, tuple):
+            if key == "keywords":
+                assert isinstance(
+                    value, (list, tuple)
+                ), f"{name} incorrectly specified GUIDE-related attribute 'keywords' (must be list or tuple)."
+            elif key == "display_name":
+                assert isinstance(
+                    value, str
+                ), f"{name} incorrectly specified GUIDE-related attribute 'display_name' (must be str)."
+            elif key == "info":
+                assert isinstance(
+                    value, str
+                ), f"{name} incorrectly specified GUIDE-related attribute 'info' (must be str)."
+            elif key == "associated_suffixes":
+                assert isinstance(
+                    value, (list, tuple)
+                ), f"{name} incorrectly specified GUIDE-related attribute 'associated_suffixes' (must be list or tuple)."
```

Comment:
Same as above; list should not be allowed (and I hope it currently isn't used anywhere)